xml/System/Half.xml

```diff
@@ -87,8 +87,8 @@
         <Parameter Name="obj" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>Compares this object to another object, returning an integer that indicates the relationship.</summary>
+        <param name="obj">An object to compare, or <see langword="null" />.</param>
+        <summary>Compares this instance to a specified object and returns an integer that indicates whether the value of this instance is less than, equal to, or greater than the value of the specified object.</summary>
         <returns>A value less than zero if this is less than <paramref name="obj" />, zero if this is equal to <paramref name="obj" />, or a value greater than zero if this is greater than <paramref name="obj" />.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Should we mention what happens if we pass `null` to the `obj` parameter?

Reply:
Good idea.